Sources/Logging/Logging.swift

```diff
@@ -138,138 +50,19 @@ extension Logging.Level: Comparable {
     }
 }
 
-extension Logging.Metadata.Value: Equatable {}
-
-/// Ships with the logging module, used to multiplex to multiple logging handlers
-public final class MultiplexLogging {
-    private let factories: [(String) -> LogHandler]
-
-    public init(_ factories: [(String) -> LogHandler]) {
-        self.factories = factories
-    }
-
-    public func make(label: String) -> LogHandler {
-        return MUXLogHandler(handlers: self.factories.map { $0(label) })
-    }
-}
-
-private class MUXLogHandler: LogHandler {
```

Comment:
Renamed this to `MultiplexLogHandler`.

Reply:
Hm, if so then `Multiplexing` sounds more correct? `MultiplexingLogHandler`, WDYT?